src/conn/conn_open.c

```diff
@@ -49,6 +49,12 @@ __wti_connection_open(WT_CONNECTION_IMPL *conn, const char *cfg[])
     /* Create the cache. */
     WT_RET(__wti_cache_create(session, cfg));
 
+    /* Initialize eviction. */
+    WT_RET(__wti_evict_create(session, cfg));
```

Comment:
There's a weak naming conflict here with `__wt_evict_create`, and it'll become a real naming conflict when we move `conn_evict.c` into the evict module.
I suggest renaming `__wt_evict_create` to something like `__wt_evict_server_create`

Reply:
Good catch, I have considered this as part of my next PR, when making `__wti_evict_create` -> `__wt_evict_create`. I will change the current `__wt_evict_create` -> `__wt_evict_threads_create`.